### ISSUE TITLE
Jhipster Registry query Metircs of microservice is failure.

### ISSUE THREAD

**Issue by yorkwu1977** [jhipster/jhipster-registry#367]
##### **Overview of the issue**

When I click [administration -  metrics] menu in jhipster-registry,  and chose to view metrics of A microservice, response error info:

{error:'invalid_token', error_description:'Cannot convert access token to JSON'}
Error: "invalid_token"
Error_description: "Cannot convert access token to JSON"

When I chose to view metrics of gateway, everything was fine.

all of micorservice, include gateway,  A microservice, uaa were generator by jhipster 6.0.0.

##### **Reproduce the error**
gateway‘s .yo-rc.json

```
{
  "generator-jhipster": {
    "databaseType": "no",
    "devDatabaseType": "no",
    "enableHibernateCache": false,
    "enableSwaggerCodegen": false,
    "enableTranslation": true,
    "jhiPrefix": "jhi",
    "languages": ["zh-cn", "en"],
    "messageBroker": false,
    "nativeLanguage": "zh-cn",
    "packageName": "com.dg.imachine.gateway",
    "packageFolder": "com/dg/imachine/gateway",
    "prodDatabaseType": "no",
    "searchEngine": false,
    "serviceDiscoveryType": "eureka",
    "skipClient": false,
    "skipServer": false,
    "testFrameworks": ["gatling"],
    "websocket": false,
    "baseName": "gateway",
    "applicationType": "gateway",
    "authenticationType": "uaa",
    "serverPort": "8080",
    "uaaBaseName": "uaa",
    "cacheProvider": "ehcache",
    "buildTool": "maven",
    "jhipsterVersion": "6.0.0",
    "skipUserManagement": false,
    "clientPackageManager": "npm",
    "clientFramework": "angularX",
    "useSass": true,
    "clientTheme": "none",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": []
  },
  "entities": ["RequestInfo", "ResponseInfo"]
}
```

A microservice's .yo-rc.json
```
{
  "generator-jhipster": {
    "databaseType": "sql",
    "devDatabaseType": "mysql",
    "enableHibernateCache": true,
    "enableSwaggerCodegen": false,
    "enableTranslation": true,
    "jhiPrefix": "jhi",
    "languages": [
      "zh-cn",
      "en"
    ],
    "messageBroker": false,
    "nativeLanguage": "zh-cn",
    "packageName": "com.dg.imachine.taxation",
    "packageFolder": "com/dg/imachine/taxation",
    "prodDatabaseType": "mysql",
    "searchEngine": false,
    "serviceDiscoveryType": "eureka",
    "skipClient": true,
    "testFrameworks": [],
    "websocket": false,
    "baseName": "taxation",
    "applicationType": "microservice",
    "authenticationType": "uaa",
    "serverPort": "8081",
    "uaaBaseName": "uaa",
    "cacheProvider": "hazelcast",
    "buildTool": "maven",
    "jhipsterVersion": "6.0.0",
    "skipUserManagement": true,
    "clientPackageManager": "npm",
    "entitySuffix": "",
    "dtoSuffix": "DTO",
    "otherModules": []
  },
  "entities": [
    "RequestInfo",
    "ResponseInfo"
  ]
}
```

##### **JHipster Registry Version(s)**

Jhipster Registry v5.0.0

##### **Browsers and Operating System**
 
chrome, edge

- [ ] Checking this box is mandatory (this is just to show you read everything)

**Comment by PierreBesson** [comment]
Please provide relevant yo-rc.json files so that we can reproduce your bug.

**Comment by PierreBesson** [comment]
Anyway I don't think this feature works for UAA.

**Comment by yorkwu1977** [comment]
I provide relevant yo-rc.json files on the top of page。
I modified jhipster-registry' boostraps-prod.yml  to use Uaa:

```
jhipster:
    http:
        cache: # Used by the CachingHttpHeadersFilter
            timeToLiveInDays: 1461
    security:
        client-authorization:
            access-token-uri: http://uaa/oauth/token
            token-service-id: uaa
            client-id: internal
            client-secret: internal
```

**Comment by PierreBesson** [comment]
Ping @xetys

**Comment by PierreBesson** [comment]
It seems to be a duplicate of #361

**Comment by yorkwu1977** [comment]
#361: response code was 500,  my issues response code was 401.

**Comment by xetys** [comment]
This should be now resolved by #392
